webapp/channels/src/components/channel_info_rhs/menu.tsx

```diff
@@ -48,11 +63,25 @@ interface MenuItemProps {
     onClick: () => void;
 }
 
+// handle keyboard activation by pressing Enter or Space.
+function handleKeyDown(e: KeyboardEvent<HTMLDivElement>, onClick: () => void) {
+    if (e.key === 'Enter' || e.key === ' ') {
```

Comment:
1. We do have key identifiers, check them out
2. I am trying to understand if we do need a wrapper or if we can just make the wrapper div not focusable at all.

Reply:
thanks for the feedback @M-ZubairAhmed .

1. thanks, I will use them.
2. I don't understand what you are suggesting here. Can you please expand?

Thanks.

Reply:
removed the key handler.